backend/model/top_container.rb

```diff
@@ -265,6 +265,33 @@ def self.batch_update(ids, fields)
   end
 
 
+  def self.bulk_update_container_profile(ids, container_profile_uri)
+    out = {}
+    cp_id = container_profile_uri.empty? ? false : JSONModel(:container_profile).id_for(container_profile_uri)
```

Comment:
Does `false` ever happen here?  Seems like the controller would have caught that case already?